app/waf/src/controllers/adc.controller.ts

```diff
@@ -36,7 +38,15 @@ export class AdcController {
     },
   })
   async create(@requestBody() adc: Partial<Adc>): Promise<Adc> {
-    return await this.adcRepository.create(adc);
+    if (!adc.id) {
```

Comment:
Should we refuse to input adc.id via API?

Reply:
Since we declare the api spec based on model definition which has all properties in it, we cannot stop user input it(generally user can remove it, or left it empty). 
if user inputs it, and in our logic, we change it to uuid or reject the request, it thus seems a bit strange. 
So, after thinking it over, I think allowing users to bring their own id is better, and at least no bad known impact.

Reply:
I set the id as not required, and put a uuid when I before the data put into DB.

Reply:
I do not think allowing user to input id when creating resource is an usual way... But we can talk that later. Since we do not have a clear API definition yet, agree to merge code first and fix it when we refining the API.